src/api/clients/IOClients/external/VTEXID.ts

```diff
@@ -19,19 +21,26 @@ export class VTEXID extends IOClient {
   }
 
   public static async invalidateBrowserAuthCookie(account: string) {
-    const configClient = ToolbeltConfig.createClient()
-    const { featureFlags } = await configClient.getGlobalConfig()
+    try {
+      const configClient = ToolbeltConfig.createClient()
+      const { featureFlags } = await configClient.getGlobalConfig()
 
-    if (featureFlags.FEATURE_FLAG_NEW_OPEN_PACKAGE) {
-      return opn(
+      if (featureFlags.FEATURE_FLAG_NEW_OPEN_PACKAGE) {
```

Comment:
I see this flag usage spread around modules. Could we have this open flow wrapped on an utility function to reduce duplication? It could be something like `switchOpen(url, opts) => { flag? open: opn...}`

Reply:
Sure, I'm going to have a call with Davi to discuss how to improve `Feature Flags estructure`, based on `Decicion Tree`(this is a option).